### PR TITLE
move icds reports task to 9:30pm IST

### PR DESCRIPTION
**Pull request by sravfeyn**
@gcapalbo

### CODE REVIEW

File: custom/icds_reports/tasks.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 celery_task_logger = logging.getLogger('celery.task')
 
 
-@periodic_task(run_every=crontab(minute=0, hour=12), acks_late=True, queue='background_queue')
+@periodic_task(run_every=crontab(minute=0, hour=16), acks_late=True, queue='background_queue')
```
Comment (gcapalbo) [inline custom/icds_reports/tasks.py]: is celery beat on icds using india timezone now, or is it still utc?

**Comment by snopoke** [comment]
Still UTC

**Comment by gcapalbo** [comment]
with https://github.com/dimagi/commcarehq-ansible/pull/798, this will now run at 4pm ist - is that ok or does it need to be moved to 21:30?

**Comment by snopoke** [comment]
Good point @gcapalbo. I'd meant to include that in the other PR.

On 15 Jun 2017 22:58, "gcapalbo" <notifications@github.com> wrote:

with dimagi/commcarehq-ansible#798
<https://github.com/dimagi/commcarehq-ansible/pull/798>, this will now run
at 4pm ist - is that ok or does it need to be moved to 21:30?

—
You are receiving this because you modified the open/close state.

Reply to this email directly, view it on GitHub
<https://github.com/dimagi/commcare-hq/pull/16629#issuecomment-308863940>,
or mute the thread

**Comment by sravfeyn** [comment]
https://github.com/dimagi/commcare-hq/pull/16665

**Pull request merged**